app/Jobs/ApplicationDeploymentJob.php

```
@@ -3187,6 +3187,19 @@ private function stop_running_container(bool $force = false)
                 $this->graceful_shutdown_container($this->container_name);
             }
         } catch (Exception $e) {
+            // If new version is healthy, this is just cleanup - don't fail the deployment
+            if ($this->newVersionIsHealthy || $force) {
+                $this->application_deployment_queue->addLogEntry(
+                    "Warning: Could not remove old container: {$e->getMessage()}",
+                    'stderr',
+                    hidden: true
+                );
+                \Log::warning("Failed to stop running container {$this->container_name}: {$e->getMessage()}");
```

PHPMD (2.15.0)

Avoid using static access to class '\Log' in method 'stop_running_container'. (undefined)

(StaticAccess)